src/main/features/core/tray.js

```diff
@@ -18,9 +18,11 @@ if (process.platform === 'darwin') {
 } else if (WindowManager.getWindowManagerGDMName() === 'kde-plasma') {
   // TODO: Change this back to ico when electron supports it on linux
   // appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main.ico`));
-  appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray.png`));
+  appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray_white.png`));
+} else if (process.platform === 'linux') {
+	appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main_tray_white.png`));
```

Comment:
Please lint before pushing to a PR, it just wastes build agent time when you don't indent correctly